pkg/util/pki/csr.go

```diff
@@ -164,9 +178,16 @@ func BuildKeyUsages(usages []v1alpha2.KeyUsage, isCA bool) (ku x509.KeyUsage, ek
 // The CSR will not be signed, and should be passed to either EncodeCSR or
 // to the x509.CreateCertificateRequest function.
 func GenerateCSR(crt *v1alpha2.Certificate) (*x509.CertificateRequest, error) {
-	commonName := crt.Spec.CommonName
-	iPAddresses := IPAddressesForCertificate(crt)
+	country := crt.Spec.X509Name.Country
 	organization := OrganizationForCertificate(crt)
+	organizationalUnit := crt.Spec.X509Name.OrganizationalUnit
+	locality := crt.Spec.X509Name.Locality
+	province := crt.Spec.X509Name.Province
+	streetAddress := crt.Spec.X509Name.StreetAddress
+	postalCode := crt.Spec.X509Name.PostalCode
+	serialNumber := crt.Spec.X509Name.SerialNumber
```

Comment:
I don't think all these vars need defining - can we just directly reference them in the struct definition below?